components/wx/impl.go

```diff
@@ -186,3 +196,43 @@ func GetAccessToken(appID string, secret string) (string, error) {
 	}
 	return body.AccessToken, nil
 }
+
+// GetWXACode ...
+func (c *Client) GetWXQrCode(token string, scene string) (image.Image, error) {
```

Comment:
build failed：
comment on exported method Client.GetWXQrCode should be of the form "GetWXQrCode ...